actionpack/lib/action_dispatch/testing/integration.rb

```diff
@@ -306,7 +306,8 @@ def kwarg_request?(*args)
         def non_kwarg_request_warning
           ActiveSupport::Deprecation.warn(<<-MSG.strip_heredoc)
             ActionDispatch::Integration::TestCase HTTP request methods will accept only
-            keyword arguments in future Rails versions.
+            the following keyword arguments in future Rails versions:
+            #{REQUEST_KWARGS.join(', ')}
```

Comment:
I was doing something like `post '/documents', document: { title: "New things", content: "Doing them" }` and saw the deprecation message. 

It was immediately clear to me what the keyword arguments were.